shared/src/commonTest/kotlin/com/tajemniktv/tajsos/data/Fakes.kt

```
@@ -42,6 +46,7 @@ class FakeCalendarProviderDao : CalendarProviderDao {
         MutableStateFlow(emptyList())
 
     override suspend fun insertProvider(provider: CalendarProviderEntity): Long = 0
+    override suspend fun insertProviders(providers: List<CalendarProviderEntity>) {}
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)